### ISSUE TITLE
Blinking dot in top left corner of editor

### ISSUE THREAD

**Issue by trailmax** [RickStrahl/MarkdownMonster#52]
Hi, thanks for this editor - it is very cool!

I've noticed a blinking dot sometimes appearing on top left corner of the editor - see animation:
![blinking-dot](https://cloud.githubusercontent.com/assets/3606356/20622200/a2b239c8-b2f9-11e6-889e-752f1ea2581c.gif)

The gif does not show it well, but it blinks with about 1sec interval. And it is there not all the time - I have not figured out the pattern when it appears.

Not an issue for me - it is barely noticeable, but usually these kinda things suggest there could be a bigger issue lurking about.

**Comment by RickStrahl** [comment]
Man that's one heck of a good eye :-) I never noticed but yeah I see it too. It looks like it's a cursor blink. Most likely from the textbox that's hidden behind the editor. I'll see what I can find...

**Comment by RickStrahl** [comment]
So I took a look and it looks like it's an artifact of the browser control. It doesn't do it in a proper browser, only in the WB control so this appears to be a quirk.

I made a hack fix for this by changing the left margin to -1 which effectively hides the pixel. It's definitely a hack, but I see nothing that's wrong with the rendering or doc display. I'll try to keep an eye on this going forward.

59c7df92b290922ce059a089495ab242e2182600

It'll be in post 1.0.14 builds.
